### ISSUE TITLE
How to Show Hover in polar chart only at middle not out side the chart circle

### ISSUE THREAD

**Issue by Prashant-Chandel** [AnyChart/AnyChart-Android#120]
Hi Team,
Please let me know how to restrict the hover to show only within the circle not outside of it in android.
Please look into image for reference 
<img width="145" alt="Screenshot_4" src="https://user-images.githubusercontent.com/17541340/64669413-192faa00-d47f-11e9-8fa9-9c891fa306f1.png">

**Comment by Shestac92** [comment]
@Prashant-Chandel 
Do you want to show the tooltip only inside the circle? Don't let it show beyond it?

**Comment by Prashant-Chandel** [comment]
@Shestac92 
Yes only inside the circle not outside of the chart circle circumference.

**Comment by Prashant-Chandel** [comment]
@Shestac92 
Thanks ,
My polar chart is not updating on data updation.
How can I update my chart to new data?

**Comment by Shestac92** [comment]
@Prashant-Chandel 
To update the chart series with new data you should make the chart active and apply new data to the existing data set. There's no need to recreate the chart.
For details, check [the comment and my answer below it](https://github.com/AnyChart/AnyChart-Android/issues/97#issuecomment-498983079). It describes how to apply new data to the data set. Absolutely the same approach is available for the polar chart.
